src/subframe_decoding.jl

```diff
@@ -381,3 +388,195 @@ function create_data(
 end
 
 
+function create_data_sub1(
```

Comment:
What would happen, if subframe 2 is the first subframe in the signal and, therefore, comes before subframe 1?
Could this still be decoded?
So imagine the subframe come in this order: `[2, 3, 4, 5, 1]`. Would that be enough or do I have to wait another round like `[2, 3, 4, 5, 1, 2, 3, 4, 5]` for a full decoding completion?

Reply:
This has not been resolved yet, has it? @kar36el

Reply:
No, you are right. We have to wait for the first subframe. But that makes for a cold start receiver totally sense. Only if we have ephemeris data from the satellite would it be usefull to decode the TOW from the subframes [2, 3, 4, 5] in your example.

Reply:
Okay does it mean, it is not possible to decode subframe 2 before subframe 1?

Reply:
In my opinion, we could however try to start decoding at each subframe, the only thing that needs to be considered is, that the IODC and IODE match for all subframes that are decoded. See chapter 20.3.4.4 in [IS-GPS-200M](https://www.gps.gov/technical/icwg/IS-GPS-200M.pdf). This could save some time required to first fix.

Reply:
See my new review on this.